interface-definitions/load-balancing-inbound-ipvsadm.xml.in

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <properties>
+      <help>Configure load-balancing</help>
+      <priority>1200</priority>
+    </properties>
+    <children>
+      <node name="inbound">
+        <properties>
+          <help>Load balancing of inbound servers</help>
+        </properties>
+        <children>
+          <node name="ipvsadm" owner="${vyos_conf_scripts_dir}/load-balancing-inbound-ipvsadm.py">
```

Comment:
Please add script to `configd-include.json`.

Also is it required to have a subnode under inbound?

Reply:
This is the idea proposed by @sever-sever , which hopes to have the expansion space to realize the second inbound load balancing in the future